index.js

```diff
@@ -1,92 +1,99 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
-
-// console.log(counter) 
+console.log("%cQuestion 1", "color: red")
+let counter = 1;
+  counter = 2;
+console.log(counter)
 // // => 2
-// console.log("%c----------", "color: red") 
-
-
+console.log("%c----------", "color: red")
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
-// *** after testing, comment the line causing the error back in 
+// *** after testing, comment the line causing the error back in
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
-
-// console.log(name)
+console.log("%cQuestion 2", "color: red")
+const name = "Da-mell"
+console.log(name)
 // => "Raffy"
-// name = "Not Raffy" 
+// name = "Not Raffy"
 // => TypeError
-// console.log("%c----------", "color: red") 
-
-
-
+console.log("%c----------", "color: red")
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
-
-// console.log(drinkWater(12))
+console.log("%cQuestion 3", "color: red")
+const drinkWater = (currentThirstLevel) => {
```

Comment:
nice use of arrow functions!! Fun fact - if there is only one parameter, then you don't need the paranthesis! In all other cases you do need them.

const drinkWater = currentThirstLevel => {